### DISCUSSION TITLE
There are 3 ways to calculate Number of H bond acceptors, Clarification needed?

### DISCUSSION THREAD

**Discussion by Kohulan** [rdkit/rdkit#6180] [Q&A]
One can calculate the number of H bond acceptors using the following,

1. Lipinski.NumHAcceptors(mol)
2. rdMolDescriptors.CalcNumHBA(mol)
3. Descriptors.NumHAcceptors(mol)

Are they equal? / what are the differences?

**Comment by Kohulan** [comment]
To calculate the Lipinski rule of 5, one can utilize the Lipinski module, which closely resembles the original paper. In alternative use cases, other Descriptor calculation functions could be utilized.

**Comment by simonmb** [comment]
I don't think the question was answered. Could someone else explain the differences?
